src/main.cpp

```diff
@@ -44,8 +44,7 @@ std::shared_ptr<Shape> shapeGirl;
 
 // Main directional light properties
 // TODO(rgarmsen2295): Move into shader manager class
-//Light curLight = {-1.75, 10, -1.75, 1, 1, 1, 0, 1};
-Light curLight = { 0, 10, 0, 1, 1, 1, 0, 1 };
+Light curLight = { -10.0f, 10.0f, 0.0f, 1.0f, 1.0f, 1.0f, 0.0f, 1.0f };

```

Comment:
This line is breaking my build. Looks like the `Light` struct has a GLint as it's last member, but here it is assigned a float.